src/Tools/AI Test Toolkit/src/Logs/AITLogEntry.Table.al

```diff
@@ -159,6 +159,21 @@ table 149034 "AIT Log Entry"
         {
             Caption = 'Output Data';
         }
+        field(40; "No. of Turns Executed"; Integer)
```

Comment:
Executed is considered a bad term. Maybe just No. of turns or No. of Turns ran or No. of Turns completed